gulpfile.js

```diff
@@ -12,6 +12,10 @@ var elixir = require('laravel-elixir');
  */
 
 elixir(function(mix) {
-    mix.less('admin.less');
-    mix.sass('main.scss');
+	mix.less('admin.less');
+	mix.sass('main.scss');
+
+	mix.copy('resources/assets/fonts', 'public/assets/fonts');
+	mix.copy('resources/assets/images', 'public/assets/images');
+	mix.copy('resources/assets/bower_components/jquery/dist', 'public/dist/js');
```

Comment:
I think a few more files are missing according to the diff: `public/dist/js/bootstrap.min.js`, `public/dist/js/Chart.min.js`, `public/dist/js/modernizr.js` 

Are those still being used?